rasa/core/actions/forms.py

```diff
@@ -688,10 +616,6 @@ async def _activate_loop(
         tracker: "DialogueStateTracker",
         domain: "Domain",
     ) -> List[Event]:
-        events = self._default_activation_events()
-
-        temp_tracker = tracker.copy()
-        temp_tracker.update_with_events(events, domain)
-        events += await self.activate(output_channel, nlg, temp_tracker, domain)
-
```

Comment:
This logic was also required to solve some reported bugs when migrating to 3.x. Please don't remove.